make/compiler/Makefile.cray-prgenv

```diff
@@ -70,4 +70,13 @@ PE_CHAPEL_PKGCONFIG_LIBS := cray-pmi:cray-ugni:$(PE_CHAPEL_PKGCONFIG_LIBS)
 endif
 endif
 
+# on login/compute nodes, lustre requires the devel api to make
+# lustre/lustreapi.h available (it's implicitly available on esl nodes)
+ifeq ($(CHPL_MAKE_AUX_FILESYS), lustre)
```

Comment:
Whoops, I'm just noticing that this should be `ifneq (,$(findstring lustre,$(CHPL_MAKE_AUXFS)))`